### PR TITLE
Modified to work with Yelp's API keys

### PR DESCRIPTION
**Pull request by tamos**
Yelp is moving to API keys, I changed a few lines to reflect that.

### CODE REVIEW

**Comment by gfairchild** [comment]
This is great! A couple of requests to complete this PR:

* Would you mind updating `requirements.txt`? `requests-oauth` and `oauthlib` should no longer be necessary with this change.
* Would you mind updating `CHANGES.md` to reflect this change? Let's bump the version to 2.2.0.
* Would you mind updating `setup.py:28` to state version 2.2.0?

File: yelpapi/yelpapi.py [hunk 1]
```diff
@@ -21,8 +21,8 @@
     THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 """
 
-from requests_oauthlib import OAuth2Session
-from oauthlib.oauth2 import BackendApplicationClient
+
```
Comment (gfairchild) [inline yelpapi/yelpapi.py]: Would you mind deleting this line?

File: yelpapi/yelpapi.py [hunk 2]
```diff
@@ -60,9 +60,12 @@ class YelpAPIError(Exception):
         """
         pass
 
-    def __init__(self, client_id, client_secret):
-        self._yelp_session = OAuth2Session(client=BackendApplicationClient(client_id=client_id))
-        self._yelp_session.fetch_token(token_url=ACCESS_TOKEN_URL, client_id=client_id, client_secret=client_secret)
+    def __init__(self, api_key):
+        ''' Instantiate a YelpAPI object.
```
Comment (gfairchild) [inline yelpapi/yelpapi.py]: Would you mind formatting these docstrings like the other docstrings in this project? That is, use double-quotes, space/indent as seen elsewhere.

**Comment by tamos** [comment]
Hi Geoff - thanks for the pointers. It's my first pull so I'm still learning.

I think I got all the points, let me know if I missed anything. I'm planning to use this package for a group project for my MA program, so thank-you for writing this and making our work easier!

- Ty

**Comment by gfairchild** [comment]
Thank you! I'm glad Yelp is moving away from OAuth. Their implementation is wonky and led to issues like #4.

One last thing - would you mind updating `examples/examples.py`? Should just require modifying lines 13, 14, and 17. I want to eventually convert those into unit tests (see #3).

**Comment by tamos** [comment]
Just updated examples.py.  I'm not too familiar with argparse, so hopefully that's all OK.

**Comment by gfairchild** [comment]
Looks good to me! Merging your code now. I'll update PyPI this afternoon/evening.

**Comment by gfairchild** [comment]
This is now live on PyPI (v2.2.1).

**Pull request merged**